webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -96,18 +98,16 @@ object DisplayNode extends Loggable {
   private[this] val removeNodeService    = RudderConfig.removeNodeService
   private[this] val asyncDeploymentAgent = RudderConfig.asyncDeploymentAgent
   private[this] val uuidGen              = RudderConfig.stringUuidGenerator
-  private[this] val nodeInfoService      = RudderConfig.nodeInfoService
   private[this] val linkUtil             = RudderConfig.linkUtil
```

Comment:
Like in previous file, the removal of `nodeInfoService` leads to simplification and some changes.